internal/workos/workos.go

```diff
@@ -2,5 +2,5 @@ package workos
 
 const (
 	// Version represents the SDK version number.
-	Version = "v4.5.0"
+	Version = "v4.6.0"
```

Comment:
Can we bump the version in a separate PR? Like we've done [here](https://github.com/workos/workos-go/pull/321) and in earlier releases.

There are other unreleased changes on `main`, so it makes the commit history easier to view in regards to what was included in each version bump.

Reply:
Yep for sure, will split.